### ISSUE TITLE
Server 500 error when publishing

### ISSUE THREAD

**Issue by liusida** [Comfy-Org/registry-backend#29]
error logs:

https://github.com/liusida/ComfyUI-Login/actions/runs/9517007745/job/26234547106

```
Validating node configuration...
Publishing node version...
{'Publisher ID is required in pyproject.toml to publish a node version'}
Error: Process completed with exit code 1.
```

**Comment by robinjhuang** [comment]
@liusida Looks like you didn't add a publisher id. It published after you added it.
https://comfyregistry.org/nodes/login

The latest run failed because you already published version 1.0.0. Need to increment the version number

**Comment by liusida** [comment]
> The latest run failed because you already published version 1.0.0. Need to increment the version number

I see. That's the issue. Thanks.
